minio/minio
# Upgrade madmin-go to 2.0.18

## Description

Upgrade madmin-go to 2.0.18

## Motivation and Context

Capture full cpu frequency stats in health info

## How to test this PR?

- Build mc with https://github.com/minio/mc/pull/4535
- Start minio server and set alias for it
- Generate health report using mc support diag {alias} --airgap
- Extract the generated report and verify that it contains a new node sys -> cpus -> freq_stats containing list of frequency stats for all the cpu cores.

## Types of changes

## Checklist:
